packages/react/src/components/Dialog/Dialog.mdx

```diff
@@ -60,6 +60,7 @@ const dialogRef = useRef<HTMLDialogElement>(null);
 ### Close on backdrop click
 
 Vi bruker `backdropClose={true}` proppen for å lukke dialogen når brukeren klikker utenfor.
```

Comment:
Kanskje siste liten, men skulle denne het `closedby` så vi aligner med det som kommer til å bli browser-standard? @Barsnes / @mimarz 
https://chromestatus.com/feature/5097714453577728

Reply:
ah, nice! 

Ja, la oss gjøre det :) 

Lag en egen PR på det?

Reply:
merged so we can create a new PR off this for name change.